clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/v3client/RegionScopedV3TitusClient.java

```diff
@@ -195,10 +198,24 @@ public void terminateJob(TerminateJobRequest terminateJobRequest) {
   @Override
   public void terminateTasksAndShrink(TerminateTasksAndShrinkJobRequest terminateTasksAndShrinkJob) {
     terminateTasksAndShrinkJob.getTaskIds().forEach(id ->
-      grpcBlockingStub.killTask(TaskKillRequest.newBuilder().setTaskId(id).setShrink(terminateTasksAndShrinkJob.isShrink()).build())
+      killTaskWithRetry(id, terminateTasksAndShrinkJob)
```

Comment:
might be worth doing some exception handling here such that the first failure doesn't stop at least attempting to destroy all the others.

Reply:
Ah great point. Will add that. Then, we can still fail at the end but only the ones with failures will have not been terminated.